src/common/SearchPage/styled.js

```diff
@@ -0,0 +1,12 @@
+import { styled } from "styled-components";
+
+export const MessageHeader = styled.h1`
+  width: 602px;
+  height: 43px;
+  font-size: 36px;
+  font-weight: 600;
+  line-height: 43.2px;
+  text-align: left;
+  color: black;
```

Comment:
we use theme features

Reply:
yeah. me too. I am front end developer too hehe